src/SprykerEco/Zed/Payone/Persistence/PayoneQueryContainer.php

```diff
@@ -21,14 +21,14 @@ class PayoneQueryContainer extends AbstractQueryContainer implements PayoneQuery
      *
      * @api
      *
-     * @param int $transactionId
+     * @param int $idPaymentPayone
```

Comment:
That looks weird
Shouldnt the interface here get fixed, or is this not possible?